api-schema/xcherry.yaml

```diff
@@ -358,8 +317,10 @@ components:
           $ref: '#/components/schemas/EncodedObject'
         commandResults:
           $ref: '#/components/schemas/CommandResults'
-        loadedGlobalAttributes:
-          $ref: '#/components/schemas/LoadGlobalAttributeResponse'
+        readAppDatabaseResponse:
```

Comment:
Why not use the name `appDatabaseReadResponse` to be almost the same as the $ref.